docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,125 @@
+# Resizing a server
+
+This guide will walk you through the required steps to change the number of CPU cores and the amount of memory your server has access to, this is done my changing the server's [flavor](/reference/flavors/).
+
+[Resize](https://docs.openstack.org/nova/latest/admin/configuration/resize.html) (or Server resize) is the ability to change the flavor of a server, thus allowing it to upscale or downscale according to user needs. A resize operation is a two-step process for the user: the first step is to perform the resize, and the second step is to either confirm (verify) success and release the old server or to declare a revert to release the new server and restart the old one.
+
+## Prerequisites
+
+You need to have a server you wish to resize. Additionally, if you prefer to work with the OpenStack CLI, then make sure to properly [enable it first](/howto/getting-started/enable-openstack-cli/).
+
+## Listing available flavors
+
+=== "{{gui}}"
+    Navigate to the server list.
+
+    ![navigation-panel](assets/resize-server/01-left-side-panel.png)
+
+    Find the server you want to resize in the list and on the right hand side click on it's menu button.
+    
+    ![menu-button](assets/resize-server/02-menu-button.png)
+
+    Click on _Modify Server_.
+
+    ![server-menu](assets/resize-server/03-menu-list.png)
+
+    On this panel near the top, find the section called _Flavor_, this is the current flavor used by the server. Press on the dropdown menu to see all available flavors.
+
+    ![server-edit](assets/resize-server/04-modify-server.png)
+=== "OpenStack CLI"
+    To list all available flavors you can simply run `openstack flavor list`, but that will return a very long unsorted list, instead we recommend the following command:
+
+    ```bash
+    openstack flavor list -c Name -f value | grep '[0-9]c[0-9]' | sort -V
+    ```
+
+    The printout is a simple and clean list, sorted by the compute type, the number of cores and then by the amount of memory.
+
+    ```plain
+    X.1c1gb
+    X.1c2gb
+    X.1c4gb
+    X.2c2gb
+    X.2c4gb
+    X.2c8gb
+    X.2c16gb
+    X.4c4gb
+    ...
+    ```
+
+## Initiating the resize
+
+Choose a new flavor that you want your server to use instead.
+
+=== "{{gui}}"
+    Once selected, the _Resize_ button will appear.
+
+    ![resize-button](assets/resize-server/05-resize-button.png)
+
+    Click on it to start the resize.
+
+    While the resize is ongoing you will see a spinning circle saying _Resize is in progress_.
+
+    ![resize-in-progress](assets/resize-server/06-resize-in-progress.png)
+=== "OpenStack CLI"
+    To start the resize use the following command:
+
+    ```bash
+    openstack server resize --flavor <new_flavor> <server_id>
+    ```
+
+    While the resize is ongoing the server should have the OS-EXT-STS:task_state of `resize_migrating` and the status of `RESIZE`.
+
+    ```bash
+    openstack server show -c OS-EXT-STS:task_state -c OS-EXT-STS:vm_state -c status <server_id>
```

Comment:
I may be wrong here, but I think non-admin accounts don't get to see the `OS-EXT-STS:foo` attributes. Please verify.

Reply:
nono, I went through these steps on my non-admin account to get these printouts.